src/Microsoft.ML.StandardTrainers/ApiCompatBaseline.txt

```diff
@@ -0,0 +1,5 @@
+Compat issues with assembly Microsoft.ML.StandardTrainers:
+MembersMustExist : Member 'Microsoft.ML.StandardTrainersCatalog.OneVersusAll<TModel>(Microsoft.ML.MulticlassClassificationCatalog.MulticlassClassificationTrainers, Microsoft.ML.Trainers.ITrainerEstimator<Microsoft.ML.Data.BinaryPredictionTransformer<TModel>, TModel>, System.String, System.Boolean, Microsoft.ML.IEstimator<Microsoft.ML.ISingleFeaturePredictionTransformer<Microsoft.ML.Calibrators.ICalibrator>>, System.Int32, System.Boolean)' does not exist in the implementation but it does exist in the contract.
```

Comment:
The recommendation here from the .NET team is to not introduce a breaking change. It breaks our guarantee of having stable APIs (see [https://devblogs.microsoft.com/dotnet/announcing-ml-net-1-0-rc-machine-learning-for-net/#updates-in-mlnet-10-rc-timeframe](https://devblogs.microsoft.com/dotnet/announcing-ml-net-1-0-rc-machine-learning-for-net/#updates-in-mlnet-10-rc-timeframe)).

If we decide we must make a breaking change, it should be done on a major version change. Please see [https://semver.org/](https://semver.org/). We should not be making an API breaking change on a minor or patch version update.

Reply:
> We should not be making an API breaking change on a minor or patch version update.

Agreed. My recommendation is to work on a process for breaking changes. We don't want to go back where .NET Framework was (i.e. no breaking changes) but we need to make it predictable for customers. My general recommendation would be: obsolete the API in a point release, leave the behavior unchanged, and remove the API in the next major version (2.x). Don't perform API- or behavior breaking changes in point releases; nobody can reason about that.